plugins/lua/stockflow.lua

```diff
@@ -527,6 +530,14 @@ function collect_reactions()
                     {job_types.MakeFlask, "Forge", "Flask"},
                     {job_types.MakeChain, "Forge", "Chain"},
                     {job_types.MakeCrafts, "Make", "Crafts"},
+                    {job_types.MakeFigurine, "Make", "Figurine"},
+                    {job_types.MakeAmulet, "Make", "Amulet"},
+                    {job_types.MakeScepter, "Make", "Scepter"},
+                    {job_types.MakeCrown, "Make", "Crown"},
+                    {job_types.MakeRing, "Make", "Ring"},
+                    {job_types.MakeEarring, "Make", "Earring"},
+                    {job_types.MakeBracelet, "Make", "Bracelet"},
+                    {job_types.MakeGem, "Make Large", "Gem"},
```

Comment:
Would "Make", "Large Gem" make more sense here? I'm not entirely sure what these two fields are (on mobile), but the item's name is "large gem".